File: 프로그래머스/1/76501. 음양 더하기/음양 더하기.java
class Solution {
    public int solution(int[] absolutes, boolean[] signs) {
        
        int sum = 0;
        
        for (int i = 0; i < absolutes.length; i++) {
            
            int val = signs[i] ? absolutes[i] : absolutes[i] * -1;
            
            sum += val;
        }
        
        return sum;
    }
}

File: 프로그래머스/1/76501. 음양 더하기/음양 더하기.kt
class Solution {
	fun solution(absolutes: IntArray, signs: BooleanArray): Int {
		var answer: Int = 0

		for ((index,bool) in signs.withIndex()) {
			if (bool) answer += absolutes[index] else answer += absolutes[index] * -1
			
		}
		
		return answer
	}
}